csharp/Facebook.Yoga.Desktop.sln

```diff
@@ -0,0 +1,102 @@
+﻿
```

Comment:
Are you building the NuGet package from the two separated solutions or just from Facebook.Yoga.sln?

---
In reply to: [92469842](https://github.com/facebook/yoga/pull/283#discussion_r92469842) [](ancestors = 92469842)